src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/Transformer.kt

```diff
@@ -38,3 +39,21 @@ fun AlertResponse.toDPSMigratedAlert(offenderNo: String) = MigrateAlertRequest(
   updatedByDisplayName = this.audit.modifyDisplayName ?: this.audit.modifyUserId,
   comments = emptyList(),
 )
+
+fun AlertResponse.toDPSMigratedAlert(latestBooking: Boolean) = MigrateAlert(
+  offenderBookId = this.bookingId,
+  // TODO get real booking sequence
+  bookingSeq = if (latestBooking) 1 else 99,
```

Comment:
There's no unique constraint including bookingSeq is there?

Reply:
Nah, they won't be storing it - they really just need to know if it is 1 or not - anyway I will be fixing that in next PR before a do migration anyway